cpp/tests/utilities/test_utilities_impl.cuh

```diff
@@ -351,5 +355,143 @@ rmm::device_uvector<value_t> mg_vertex_property_values_to_sg_vertex_property_val
   return sg_values;
 }
 
+template <typename vertex_t, bool multi_gpu>
+rmm::device_uvector<vertex_t> randomly_sample_vertices(
+  raft::handle_t const& handle,
+  raft::random::RngState& rng_state,
+  std::vector<vertex_t> const& vertex_partition_range_lasts,
+  size_t sample_size,
+  bool with_replacement,
+  bool sort_samples)
+{
```

Comment:
Should we perhaps use https://github.com/rapidsai/cugraph/blob/60bd0eb0a5860deda1c440a105be0781c0afec31/cpp/include/cugraph/graph_functions.hpp#L903, or combine these by adding the `sort_samples` parameter to that function and perhaps using this implementation?  Seems like the functionality should be the same.

Reply:
Oh, yes, you're right. I forgot about the function.